modules/instrument_manager/php/NDB_Menu_Filter_instrument_manager.class.inc

```diff
@@ -93,7 +94,7 @@ class NDB_Menu_Filter_instrument_manager extends NDB_Menu_Filter
                 } else if($key == "Table_Installed") {
                     // This should also check that all the columns exist and have the right type, for new style instruments
                     $exists = $db->pselectOne("SELECT count(*) FROM information_schema.tables WHERE TABLE_SCHEMA=:dbname AND TABLE_NAME=:tablename", array(
-                        'dbname' => 'loris',
+                        'dbname' => $config->getSetting('database')['database'],
```

Comment:
I don't think this will work in some older versions of PHP (5.3, maybe? I think directly indexing an array from a return value was only added to PHP in 5.4, but I might be getting my versions mixed up)..Settings->dbName() should directly return the database name and doesn't require any array indexing.